dataactvalidator/config/sqlrules/fabs31_5.sql

```diff
@@ -32,6 +31,29 @@ WHERE NOT EXISTS (
                  END) >= CAST(sam_recipient.registration_date AS DATE)
             AND (CASE WHEN is_date(COALESCE(fabs.action_date, '0'))
                  THEN CAST(fabs.action_date AS DATE)
-                 END) < CAST(sam_recipient.expiration_date AS DATE)
-        )
-    );
+                 END) < CAST(sam_recipient.expiration_date AS DATE)))),
+special_exception_{0} AS
+    (SELECT *
+    FROM active_recipient_{0} AS fabs
+    WHERE NOT (
+         UPPER(legal_entity_country_code) <> 'USA'
+         AND (CASE WHEN is_date(COALESCE(action_date, '0'))
+              THEN CAST(action_date AS DATE)
+              END) > CAST('10/01/2024' AS DATE)
+         AND EXISTS (
+            SELECT 1
+            FROM sam_recipient_unregistered AS sru
+            WHERE UPPER(fabs.uei) = UPPER(sru.uei))))
+SELECT
+    row_number,
+    action_date,
+    action_type,
+    legal_entity_country_code,
+    uei,
+    afa_generated_unique AS "uniqueid_AssistanceTransactionUniqueKey"
+FROM special_exception_{0} AS fabs
+WHERE NOT (
+          (CASE WHEN is_date(COALESCE(action_date, '0'))
+           THEN CAST(action_date AS DATE)
+           END) < CAST('01/01/2017' AS DATE)
+           AND UPPER(COALESCE(correction_delete_indicatr, '')) = 'C');
```

Comment:
Was this intentionally left in? I see nothing in any of the rule text that indicates 2017 matters. I know there was a discussion about this but it sounded like the decision was not to include it?

Reply:
No purpose, just being consistent with the others. Agreed it should be removed.